mitodl/open-discussions
# search styles

#### Pre-Flight checklist

  
<img width="1096" alt="Screen Shot 2022-08-09 at 8 06 10 PM" src="https://user-images.githubusercontent.com/1934992/183788321-6f07de57-905e-4859-a5b1-e0f151c8c522.png">
<img width="1097" alt="Screen Shot 2022-08-09 at 8 06 32 PM" src="https://user-images.githubusercontent.com/1934992/183788330-3ae08c39-1f37-4e8f-a819-9142a582a805.png">
<img width="376" alt="Screen Shot 2022-08-09 at 8 07 11 PM" src="https://user-images.githubusercontent.com/1934992/183788325-90900527-6b9b-4c30-b73b-7968ae91c1d6.png">
<img width="376" alt="Screen Shot 2022-08-09 at 8 07 04 PM" src="https://user-images.githubusercontent.com/1934992/183788327-60ed7901-b154-4a08-864a-15bdc1f69197.png">
<img width="372" alt="Screen Shot 2022-08-09 at 8 06 57 PM" src="https://user-images.githubusercontent.com/1934992/183788328-8057953e-57c2-4561-bdeb-33682890c36e.png">



#### What are the relevant tickets?
https://github.com/mitodl/open-discussions/issues/3595

#### What's this PR do?
This pr fixes the styling for infinite corridor search on both desktop and mobile

#### How should this be manually tested?
Go to /infinite/search. Verify that the desktop site looks like the design at https://projects.invisionapp.com/d/main#/console/22082155/468407665/preview
There isn't a mobile design, so just verify that the mobile site isn't obviously broken
